src/database/mock.rs

```diff
@@ -204,6 +204,33 @@ where
     }
 }
 
+impl<M, N> IntoMockRow for (M, N)
+where
+    M: ModelTrait,
+    N: ModelTrait,
+{
+    fn into_mock_row(self) -> MockRow {
+        let mut mapped_join = BTreeMap::new();
+
+        for column in <<M as ModelTrait>::Entity as EntityTrait>::Column::iter() {
+            mapped_join.insert(format!("A_{}", column.as_str()), self.0.get(column));
+        }
+        for column in <<N as ModelTrait>::Entity as EntityTrait>::Column::iter() {
+            mapped_join.insert(format!("B_{}", column.as_str()), self.1.get(column));
```

Comment:
We should use `SelectA` & `SelectB` here instead of manually writing it.
- https://github.com/SeaQL/sea-orm/blob/0f164b62d5c4b36d9040d4ef933eeb1cb97a7359/src/query/combine.rs#L31-L32